edgeless_orc/src/orchestrator/test.rs

```diff
@@ -26,6 +26,8 @@ enum MockAgentEvent {
     PatchResource(edgeless_api::common::PatchRequest),
```

Comment:
Skipped reviewing this file due to time-constraints.

Reply:
In general, you don't want to review unit tests...